lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -16,9 +17,9 @@
     protected $_name;
 
     /**
-     * @var string the text for this suggestion
+     * @var string the prefix for this suggestion
      */
-    protected $_text;
+    protected $_prefix;
```

Comment:
Ok no I think to have a more clear overview of the problem :

in ES we have in [SuggestBuilder](https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/search/suggest/SuggestionBuilder.java#L56) these params :

- protected static final ParseField TEXT_FIELD = new ParseField("text");
- protected static final ParseField PREFIX_FIELD = new ParseField("prefix");
- protected static final ParseField REGEX_FIELD = new ParseField("regex");
- protected static final ParseField FIELDNAME_FIELD = new ParseField("field");
- protected static final ParseField ANALYZER_FIELD = new ParseField("analyzer");
- protected static final ParseField SIZE_FIELD = new ParseField("size");
- protected static final ParseField SHARDSIZE_FIELD = new ParseField("shard_size");

I think we should follow what ES is doing, prefix and text still exists in all latest branches "5.0/5.x/master ".

Anyway
we are dealign with the :
- field 
- size
- shard_size
with a **setParam**

we are dealign with the :
- text
- prefix

with a **property**

I think the idea of @ruflin is the more smoth and logic to use PARAMS; and maybe we are loosing analyzer :)  

what do you think guys ? @ruflin @Zyqsempai ?